File: ympa_fsharp_functions_server/src/model/CategoryErrorDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.CategoryErrorType

module CategoryErrorDTO =

  //#region CategoryErrorDTO

  [<CLIMutable>]
  type CategoryErrorDTO = {
    [<JsonProperty(PropertyName = "categoryId")>]
    CategoryId : int64;
    [<JsonProperty(PropertyName = "type")>]
    Type : CategoryErrorType;
  }

  //#endregion

File: src/main/scala/org/openapitools/server/model/CategoryErrorDTO.scala
package org.openapitools.server.model


/**
 * Текст ошибки.
 *
 * @param categoryId Идентификатор категории. for example: ''null''
 * @param `type`  for example: ''null''
*/
final case class CategoryErrorDTO (
  categoryId: Option[Long] = None,
  `type`: Option[CategoryErrorType] = None
)

